jjb/egeria/egeria.yaml

```diff
@@ -1,4 +1,13 @@
 ---
+- job-group:
```

Comment:
This entire section isn't needed (at this time) It only makes sense if you are going to use this job group more than once.